pom.xml

```diff
@@ -64,6 +64,30 @@
                 <artifactId>sonar-maven-plugin</artifactId>
                 <version>3.7.0.1746</version>
             </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>0.8.7</version>
```

Comment:
Latest version is 0.8.9
A good way to know which version an artifact has is through https://mvnrepository.com/
For example: jacoco-maven-plugin: https://mvnrepository.com/artifact/org.jacoco/jacoco-maven-plugin